runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -672,4 +672,15 @@ default boolean swapTemporossLeave()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "swapCropProtection",
+		name = "Crop Protection",

```

Comment:
This isn't very clear without mentioning that it actually applies to flowers. I'd make this change for the config name, but everything else should be fine.
```suggestion
		name = "Crop Protecting Flowers",

```

Reply:
I went with flowers at first but was unsure if that's technically fitting since it applies to scarecrows too. Should it still say flowers?